src/index.html

```diff
@@ -6,9 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo_container">
+        <a href="index.html" class="logo_link">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="index.html" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="index.html" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="index.html" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="index.html" class="nav_link" data-qa="hover">Laptops & Computers</a>
```

Comment:
please fix the code styling according to the checklist 
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/815c0b5d-8752-42a3-b3a0-48a0e0815cba)